fs-api-blockio/src/System/FS/BlockIO/IO.hs

```diff
@@ -0,0 +1,17 @@
+module System.FS.BlockIO.IO (
+    ioHasBlockIO
+  , I.IOCtx
+  , I.IOCtxParams (..)
+  ) where
+
+import           System.FS.BlockIO.API
+import qualified System.FS.BlockIO.Internal as I
+import           System.FS.IO
+
+ioHasBlockIO :: HasBlockIO IO HandleIO I.IOCtxParams I.IOCtx
```

Comment:
If we use a generic impl of `HasBlockIO` for the serial case then this is going to need to be passed the `HasFS`. And that makes sense anyway, since it's supposed to be an API extension of `HasFS`.